src/python/pants/backend/python/dependency_inference/parse_python_dependencies.py

```diff
@@ -31,21 +32,37 @@ class ParsedPythonImports(FrozenDict[str, ParsedPythonImportInfo]):
     """All the discovered imports from a Python source file mapped to the relevant info."""
 
 
+class ParsedPythonAssets(DeduplicatedCollection[str]):
```

Comment:
Maybe `ParsedPythonAssetPaths`? I continue to confuse myself with what this is, I think because of the old setup of it being a tuple.

Reply:
Yeah. If/when we support `pkgutil.get_data` we'll be changing this name :sweat_smile: but for now that works!